tests/unit/Makefile.am

```diff
@@ -285,37 +283,34 @@ avahi_config_test_SOURCES = avahi_config_test.c \
 	../../cf-serverd/server_access.c \
 	../../cf-serverd/server_classic.c \
 	../../cf-serverd/strlist.c
-avahi_config_test_LDADD = ../../libpromises/libpromises.la libtest.la
 
 endif
 endif
 
 #file_writer_test_CPPFLAGS = -I$(top_srcdir)/libutils
-file_writer_test_SOURCES = file_writer_test.c gcov-stub.c
+file_writer_test_SOURCES = file_writer_test.c
```

Comment:
This doesn't seem like a related change to me.

Reply:
Sure. I can move this to another PR. I was unable to run unit tests locally with gcov-stub.c present. Maybe it runs OK in some cloud situations since the point of gcov-stub.c is to handle the case where the codebase has gcov enabled but the test code does not.